### PR TITLE
AR-5721 - Update documentation for new appointment endpoint to request an appointment be cancelled

### PR DESCRIPTION
**Pull request by jsmet**
@VinnieMaida @JakeDriscoll @BobCardillo @joerocca

### CODE REVIEW

File: source/includes/_appointment.md [hunk 1]
```diff
@@ -222,6 +222,40 @@ PUT https://select.nextech-api.com/api/Appointment/5453
...
+`PUT /Appointment/{identifier}` 
+
+
+#### Parameters
+| Name | Located in | Description | Required | Initial Version |
+| ---- | ---------- | ----------- | -------- | --------------- |
+| identifier | path | The unique identifier of the appointment to be updated | Yes | _14.6_ |
+| appointment | body | The appointment object representing the appointment that is being requested to be cancelled | Yes | _14.6_ |
+
+The commit parameter should be in the form of an Appointment status field of `cancelled` with a custom extension url of `https://select.nextech-api.com/api/structuredefinition/request-type` and value of `queued`. See [https://www.hl7.org/fhir/http.html#update](https://www.hl7.org/fhir/http.html#update) for details on formatting PUT requests in RESTful API’s.
```
Comment (VinnieMaida) [inline source/includes/_appointment.md]: The value actually has to be `queue`, not `queued` .

Also, should there be a blurb in this section about not being able to request cancellations for appointments that don't exist, are already cancelled, or have already been requested for cancellation? Probably kind of common sense, just not sure if we need to explicitly note it here or not.

**Review by VinnieMaida — CHANGES_REQUESTED** [review]
Noticed one really small typo, and had a question.

**Review by VinnieMaida — APPROVED** [review]
Looks good. The error messages given back by this endpoint are meaningful, so I don't really see an explicit need to list out those error cases here in the documentation.

**Comment by BobCardillo** [comment]
@dustinbrinegar fyi as well in case you aren't monitoring this repo yet.

**Pull request merged**